quality-assurance/http-api-mocking-tools.md

```diff
@@ -0,0 +1,52 @@
+# Mocking HTTP Endpoints
+
+This section covers tools in use for creating stub/mock HTTP endpoints/APIs such as for testing a deployed system or testing a service in development in the absence of another required real service.
+
+Notable tools which you can use as an alternative to building your owm mock http APIs are:
+
+  - [Wiremock](http://wiremock.org/)
+  - [Prism](https://stoplight.io/open-source/prism) (Not yet investigated)
+
+## Wiremock
+
+### Simple Mocks
+
+Use wiremock to simplify creating mocks/stubs _unless_ your requirements:
+
+- Are beyond fixed or templated HTTP responses and callbacks (including to other URIs) and basic unhappy path tests, most of which Wiremock can cover.
+- Would make a Wiremock based solution more effort or complexity than creating a bespoke mock.
+
+Many simple HTTP endpoint stubbing use cases are solved problems for wiremock.
+
+#### Use Cases
```

Comment:
I think the "Use Cases" and "Versions"  section should be merged into one.  i.e. there are two use cases: Deployed instances (use Java) and inline (use the .net version)

Reply:
Thanks - Have done so